### ISSUE TITLE
keyboard shortcuts: collapse tree and create spline

### ISSUE THREAD

**Issue by zozorg** [synfig/synfig#491]
**Issue description**:
It would be nice to have keyboard shortcuts to : 
- collapse all groups/sets
- collapse one level of group/set ( collapse above the currently selected layer/set )
- finish a spline ( the "enter" key could finish the spline, just like "esc" discards it )

**Comment by morevnaproject** [comment]
Hello! WHich chortcuts would you like to have for first two items?
For last item (about spline) - can you please put it into separate request? ^__^

**Comment by morevnaproject** [comment]
Not e to myself: could be blocked by https://github.com/synfig/synfig/issues/303

**Comment by zozorg** [comment]
I think "left" and "right" would be intuitive to "collapse one level" and "expand selected group layer".
Collapse all could then be ctrl + left

**Comment by Svarov-RZM** [comment]
+1 to this one.
It would be even better to implement it not just for groups/sets, but for all expandable properties like 'Vertices' list so you can navigate through different vertex properties and custom conversions.

**Comment by rodolforg** [comment]
I think this issue was closed by mistake…

**Comment by morevnaproject** [comment]
Yeah, accidentally closed this by pushing commit. Thank you!

**Comment by rodolforg** [comment]
It seems GTK uses plus `+` and minus `-` keys for tree item collapse and expansion.
Pressing theses keys with `shift` reveal/hide all its children.

**Comment by rodolforg** [comment]
@zozorg  could you confirm these (GTK) shortcuts mentioned in previous comment?

**Comment by zozorg** [comment]
They do work, but "minus" doesn't behave like I described ; it only collapses the currently selected group layer. What I was looking for was to collapse the group layer in which the currently selected layer is located (and move selection to the now collapsed group layer). 
If the currently selected layer is a group layer, I think the intuitive behaviour would be : if it is expanded, collapse it ; if it is already collapsed, collapse one level above (same behaviour as with non-group layer).
Using left and right arrows also would make navigation easier than using plus and minus.

**Comment by zozorg** [comment]
I'm not sure I should have put a bounty on this one, since I'm not the one deciding how the UI should behave ^^'
@rodolforg (or anyone else involved in Synfig development, actually) Do you have interest in ethereum bounties? I used to set USD bounties, but somehow Bountysource removed the "pay bitcoin, create USD bounty" option, which was what I was using...

**Comment by zozorg** [comment]
I finally can add USD bounties again, but I can't put one on that issue because there is already an ethereum bounty offered .... somehow you can't have both USD and ETH.
Setting an unofficial $50 bounty, I can add a "fake" bounty on whatever other issue you are about to resolve.

**Comment by morevnaproject** [comment]
Since BountySource is giving so ugly hiccups, I actually wanted to try https://issuehunt.io/

What people think about it?

**Comment by nattymbt** [comment]
Could I take this issue?
